src/main/java/pl/nogacz/snake/board/Board.java

```diff
@@ -83,10 +88,25 @@ private void moveSnake() {
         }
     }
 
-    private void moveSnakeHead(Coordinates coordinates) {
+    private void moveSnakeHead(Coordinates coordinates) { // his head will be at foods location
         if(coordinates.isValid()) {
+            if(rottenFoodCoordinates == null){ // There is no rotten food on the field
+                int randomnum = random.nextInt(10);
+                if( randomnum == 1 ){ // Randomness
+                    addEat(false);
+                } 
+            }
+            else{   
+                System.out.println(counter);
+                if(counter < 20) counter++;
+                if(counter == 20){ // Disappear after 20 snake moves.

```

Comment:
20 değeri için bir constant tanımla ve constantı kullan.

Reply:
Halledildi